pkg/credentials/credentials.go

```diff
@@ -37,6 +38,24 @@ func MakeAwsEnvars(cfg *projectconfig.ZeroProjectConfig, awsSecrets Secrets) []s
 	return env
 }
 
+func AwsCredsPath() string {
+	usr, err := user.Current()
+	if err != nil {
+		log.Fatal(err)
+	}
+	return filepath.Join(usr.HomeDir, ".aws/credentials")
+}
+
+func FillAWSProfileCredentials(credsPath string, profileName string, creds globalconfig.ProjectCredential) globalconfig.ProjectCredential {
```

Comment:
Maybe just "Get" rather than "Fill"?

Reply:
Also, maybe split into two functions, one of which doesn't accept the path and just calls the other with the default of `AwsCredsPath()`? Seems like most likely the only time you would provide a different path would be for tests.